exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillPushProjectIntoScanRule.java

```diff
@@ -61,12 +61,12 @@ protected boolean skipScanConversion(RelDataType projectRelDataType, TableScan s
   public static final RelOptRule DRILL_LOGICAL_INSTANCE =
       new DrillPushProjectIntoScanRule(LogicalProject.class,
           DrillScanRel.class,
-          "DrillPushProjIntoDrillRelScan");
+          "DrillPushProjectIntoScanRule");
```

Comment:
Should this be DrillPushProjectIntoDrillRelScan?

Reply:
No, it is useful to have a description of the rule which contains the name of the class to find the rule reading the logs for example. So I will change it to `DrillPushProjectIntoScanRule:logical`.

Reply:
Yes adding the word "logical" will help differentiate between logical and physical.